obspy/taup/tau.py

```diff
@@ -4,6 +4,12 @@
                         unicode_literals)
 from future.builtins import *  # NOQA
 
+import copy
+
+import matplotlib.cbook
+import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
This is still a bit confusing to me. I like the general structure future -> stdlib -> third party -> obspy but I think each block in itself should be alphabetically ordered. The numpy -> matplotlib -> scipy order and all at the end of the third party block makes little sense to me...but I value consistency more than this so I guess I'll change it ;-)